### DISCUSSION TITLE
top filter sometimes it doesn't work on cell phones

### DISCUSSION THREAD

**Discussion by redati** [breezefront/community#295] [1. Help]
nice project, I'm trying to investigate an error that occurs sometimes, I try to click on the top filter using mobile, but it doesn't open the filter menu.

Has anyone come across this?

latest magento and module, theme evolution

![Captura de tela 2024-06-28 115645](https://github.com/breezefront/community/assets/6043222/63b469a5-6c68-464f-90d9-7e7866cfbc2f)

![Captura de tela 2024-06-28 115623](https://github.com/breezefront/community/assets/6043222/daa86580-cbbe-4f4b-bc11-3a901036a92d)
![Captura de tela 2024-06-28 115634](https://github.com/breezefront/community/assets/6043222/973c3ba1-10da-4f3c-957c-ca4070816a22)

**Comment by vovayatsyuk** [comment]
Please provide a URL to your site.

**Reply by redati** [reply]
> Please provide a URL to your site.
https://www.decoraonline.com.br/quadros-decorativos/quadros-reproducao-pintores-famosos

I'm investigating localhost, without the amasty module, it still occurs, especially if you load the page and click on the filter immediately.

**Reply by vovayatsyuk** [reply]
Sorry, I can't reproduce this issue. Can you please describe how to reproduce it?
